evm/v0.5/contracts/dev/PrepaidAggregator.sol

```diff
@@ -386,11 +416,28 @@ contract PrepaidAggregator is AggregatorInterface, Ownable, WithdrawalInterface
 
   function deleteRound(uint32 _id)
     private
-    onlyIfMaxAnswersReceived(_id)
+    ifMaxAnswersReceived(_id)
   {
     delete rounds[_id].details;
   }
 
+  function timedOut(uint32 _id)
+    private
+    returns (bool)
+  {
+    if (_id <= 1) { // must have at least one round of history
+      return false;
+    }
+    return rounds[_id - 1].updatedTimestamp < block.timestamp - timeout;
```

Comment:
I'm not sure if we're properly testing this. When I remove the `- 1`, all the tests still pass:
```
    return rounds[_id].updatedTimestamp < block.timestamp - timeout;
```
The input for this function is always a previous ID which has already been decremented. So this is really the round's `_id - 2`.